Source/Facebook/FacebookClient.cs

```diff
@@ -602,7 +602,10 @@ private HttpHelper PrepareRequest(HttpMethod httpMethod, string path, object par
                     throw new ArgumentOutOfRangeException("httpMethod");
             }
 
-            request.ContentType = contentType;
+            if (contentType != null)

```

Comment:
could u use `string.IsNullOrEmpty` similar to the etag one. Also remove the braces.